scripts/lock-translations.py

```diff
@@ -25,7 +25,7 @@ def get_local_resources(tx_config, project):
     config.read(tx_config)
     
     try:
-      project in re.search('p:python-[\w]+:', config.sections()[1]).group(0)
+      project in re.search('p:python-\w+:', config.sections()[1]).group(0)
```

Comment:
```suggestion
      project in re.search(r'p:python-\w+:', config.sections()[1]).group(0)
```
We need `r` prefix for string or double backslash not to let Python interpret `\w` as an escape sequence.

Alternative:
```suggestion
      project in re.search('p:python-\\w+:', config.sections()[1]).group(0)
```

Reply:
See https://docs.python.org/3/reference/lexical_analysis.html#escape-sequences